src/org/jetbrains/java/decompiler/modules/decompiler/sforms/SSAUConstructorSparseEx.java

```diff
@@ -98,9 +96,11 @@ public void splitVariables(RootStatement root, StructMethod mt) {
 
     ssuversions.initDominators();
 
-    DotExporter.toDotFile(ssuversions, mt, "ssauVarVer");
+    DotExporter.toDotFile(ssuversions, mt, "ssauVarVer" + xxx++, varAssignmentMap);
   }
 
+  static private int xxx = 0;
```

Comment:
Old debug code

Reply:
This is to prevent the dot files from overriding eachother. Should I give it a better name, or just remove it?

Reply:
I think you should remove it for now